basketball_reference_scraper/request_utils.py

```diff
@@ -0,0 +1,16 @@
+from requests import get
+from time import sleep
+
+def get_wrapper(url):
+    r = get(url)
+    while True:
```

Comment:
simpler variant of what you wrote: `while r.status_code != 200`
but this way you cannot return anything that is not 200, and catch errors.
so, probably best:
`while r.status_code == 429: ...`